lib/exop/validation_checks.ex

```diff
@@ -116,7 +116,8 @@ defmodule Exop.ValidationChecks do
     if check_item_present?(check_items, item_name) do
       check_item = get_check_item(check_items, item_name)
 
-      TypeValidation.check_value(check_item, check) || %{item_name => "has wrong type"}
+      TypeValidation.check_value(check_item, check) ||
+        %{item_name => "has wrong type. expected type: #{check}, got: #{inspect(check_item)}"}
```

Comment:
here you have a comma, in all places below - a dot. I would say a comma looks more natural if you have one-sentence error message :)